prisma/schema.prisma

```diff
@@ -1,6 +1,5 @@
 generator client {
-  provider      = "prisma-client-js"
-  binaryTargets = ["native", "rhel-openssl-1.0.x"]
```

Comment:
This has to stay, or else it won't work on AWS

Reply:
sure! I think it was autogenerated